src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/ApprovedPremisesSeedJob.kt

```diff
@@ -36,6 +36,25 @@ class ApprovedPremisesSeedJob(
     probationRegion = columns["probationRegion"]!!,
     localAuthorityArea = columns["localAuthorityArea"]!!,
     characteristics = columns["characteristics"]!!.split(",").filter { it.isNotBlank() }.map { it.trim() },
+    isIAP = parseBooleanStringOrThrow(columns["isIAP"]!!, "isIAP"),
```

Comment:
Could we not type these as Boolean in ApprovedPremisesSeedCsvRow and return a Boolean from `parseBooleanStringOrThrow` ?